tests/assertions.ts

```diff
@@ -50,4 +51,48 @@ export async function assertGeneratedCorrectly({
 
     expect(await fse.pathExists(pathToFile), `${pathToFile} exists`).toBe(true);
   }
+
+  await matchesFixture('.prettierrc.js', { cwd: projectRoot });
+}
+
+export async function matchesFixture(
+  /**
+   * Project-relative file to test against
+   */
+  testFilePath: string,
+  options?: {
+    /**
+     * Which fixture set to use
+     */
+    scenario?: string;
+    /**
+     * By default, the file used will be the same as the testFilePath, but
+     * in the fixtures directory under the (maybe) specified scenario.
+     * this can be overridden, if needed.
+     * (like if you're testFilePath is deep with in an existing monorepo, and wouldn't
+     *   inherently match our default-project structure used in the fixtures)
+     */
+    file?: string;
+
+    /**
+     * The working directory to use for the relative paths. Defaults to process.cwd() (node default)
+     */
+    cwd?: string;
```

Comment:
do you forsee needing this? 🤔 it seems like the stuff that you're doing to the fixtureFile won't find the right file if you go relative to cwd 🤔 I can understand maybe needing it for the test file, but the fixture files should always be pretty much in the same place right? and I don't see the direct correlation between changing the cwd on the fixtures and the output at the same time 🤔

Reply:
> do you forsee needing this? 

yes, specifically for when we test generating in an existing monorepo, but want to re-use _parts_ of a fixture within the deeper monoreepo

> but the fixture files should always be pretty much in the same place right?

nay, see above

Reply:
I can demonstrate more clearly in this PR:https://github.com/embroider-build/addon-blueprint/pull/154